simexpal/base.py

```diff
@@ -522,6 +533,55 @@ def make_variation(prod):
 
 		return [make_variation(prod) for prod in itertools.product(*variation_bundle)]
 
+	def update_status_cache_dict(self, run):
+
+		def _get_status_entry(run):
```

Comment:
Since this function is only called once, we can inline it into `update_status_cache_dict`.

Reply:
done